helm/superset/templates/init-job.yaml

```diff
@@ -37,6 +41,9 @@ spec:
       {{- if .Values.init.podAnnotations }}
       annotations: {{- toYaml .Values.init.podAnnotations | nindent 8 }}
       {{- end }}
+      {{- if .Values.init.podLabels }}
+      labels: {{- toYaml .Values.init.podLabels | nindent 8 }}
+      {{- end }}
```

Comment:
Actually, a recent PR #30710 introduced the concept of `extraLables`. Can we change this so that both `.Values.init.podLabels` and `.Values.extraLabels` are considered here?

Reply:
I noticed that but it was unclear to me if only the Job should receive the labels or the child pod should still get them. Anyway, the last commits adds that.

Reply:
Yes, the last commit implements them extactly as intended, thanks for the quick update 👍